src/background.cpp

```diff
@@ -1,4 +1,6 @@
 #include <cmath>
+#include <memory>
```

Comment:
`make_unique` is defined in the same place as `unique_ptr`, there is no need in additional includes.